eventstore/sql/create_events.sql

```diff
@@ -66,6 +66,30 @@ INSERT INTO events with
 			where
 				raw_message->>'service_instance_type' = 'managed_service_instance'
 				and raw_message->>'space_name' !~ '^(SMOKE|ACC|CATS|PERF)-' -- FIXME: this is open to abuse
+		) union all (
+			select
+				id as event_sequence,
+				guid::uuid as event_guid,
+				created_at,
+				(raw_message->>'task_guid')::uuid as resource_guid,
+				(raw_message->>'task_name') as resource_name,
+				'task'::text as resource_type,                              -- resource_type for compute resources
+				(raw_message->>'org_guid')::uuid as org_guid,
+				(raw_message->>'space_guid')::uuid as space_guid,
+				'ebfa9453-ef66-450c-8c37-d53dfd931038'::uuid as plan_guid,  -- plan guid for all compute resources
+				'task'::text as plan_name,                                  -- plan name for all compute resources
```

Comment:
Should these comments say `for all task resources`?

Reply:
Good catch :D